src/api/browserview/browserview.ts

```diff
@@ -41,7 +41,17 @@ export interface BrowserViewCreationOptions extends BrowserViewOptions {
     };
 }
 
+/**
+ * @lends View
+ */
 export class BrowserViewModule extends Base {
+    /**
+     * Creates a new View.
+     * @param { View~options } options - View creation options
+     * @return {Promise.<View>}
+     * @tutorial View.create
+     * @static
+     */
```

Comment:
The return type is missing in this function.

Reply:
the return type is defined there with `@return {Promise.<View>}`